src/fidesops/ops/api/v1/endpoints/masking_endpoints.py

```diff
@@ -23,19 +24,38 @@
 
 @router.put(MASKING, response_model=MaskingAPIResponse)
 def mask_value(request: MaskingAPIRequest) -> MaskingAPIResponse:
-    """Masks the value(s) provided using the provided masking strategy"""
+    """Masks the value(s) provided using the provided masking strategies
+
+    Each masking strategy supplied will be run on all the values in sequence.  The masked values
+    from one masking strategy will be passed into the next masking strategy.
+
+    """
     try:
-        values = request.values
-        masking_strategy = request.masking_strategy
-        strategy = MaskingStrategy.get_strategy(
-            masking_strategy.strategy, masking_strategy.configuration
-        )
-        logger.info(
-            "Starting masking of %s value(s) with strategy %s",
-            len(values),
-            masking_strategy.strategy,
-        )
-        masked_values = strategy.mask(values, None)
+        values = request.values or []
+        masked_values: Optional[List[Any]] = request.values.copy()
```

Comment:
Should this be optional? It looks like it might always be an empty list if nothing else?

Reply:
yes I agree with you, I was getting  "Incompatible types in assignment" mypy errors because `masking.mask` is allowed to take in None.  I think we should instead ignore the type hint error then when we call `masking.mask`